x/evmutil/keeper/conversion_evm_native_bep3.go

```diff
@@ -0,0 +1,38 @@
+package keeper
+
+import (
+	"math/big"
+
+	"github.com/kava-labs/kava/x/evmutil/types"
+)
+
+var (
+	DefaultBEP3ConversionDenoms = []string{"bnb", "busd", "btcb", "xrpb"}
```

Comment:
nit, I'd make this variable private (or use constants) to avoid other packages being able to modify it. Low chance of it being a problem but adding unauthorised assets here would allow minting of coins

Reply:
Good catch @rhuairahrighairidh